interactive/rabbit_mq_model_runner.py

```diff
@@ -0,0 +1,68 @@
+# main model runner interface class for rabbitmq and actual model runner
+# 1) listen to run_queue
```

Comment:
Why is this stuff duplicated here?

Reply:
Couldn't you just make this into a reusable thing that you can import rather than having the same code twice?

Reply:
not sure I get what you mean, this code is only used once. there's another file that listens to a different queue if that's what you're referring to